### PR TITLE
Update FleetReSortie.cs

### PR DESCRIPTION
**Pull request by thethingexe**
Sinks can't sink at moderate damage, so make it notify at severely damaged. Which was what was happening in Musashi's release.

### CODE REVIEW

File: Grabacr07.KanColleWrapper/Models/FleetReSortie.cs [hunk 1]
```diff
@@ -1,171 +1,171 @@
...
-			if (ships.Length == 0)
-			{
-				this.ReadyTime = null;
-				this.UpdateCore();
-				return;
-			}
-
-			var reason = CanReSortieReason.NoProblem;
-
-			if (ships.Any(s => (s.HP.Current / (double)s.HP.Maximum) <= 0.5))
```
Comment (thethingexe) [inline Grabacr07.KanColleWrapper/Models/FleetReSortie.cs:111]: *

File: Grabacr07.KanColleWrapper/Models/FleetReSortie.cs [hunk 2]
```diff
@@ -1,171 +1,171 @@
...
+			if (ships.Length == 0)
+			{
+				this.ReadyTime = null;
+				this.UpdateCore();
+				return;
+			}
+
+			var reason = CanReSortieReason.NoProblem;
+
+			if (ships.Any(s => (s.HP.Current / (double)s.HP.Maximum) <= 0.25))
```
Comment (thethingexe) [inline Grabacr07.KanColleWrapper/Models/FleetReSortie.cs:111]: Only lined changed, not sure why it's being weird.

**Comment by thethingexe** [comment]
Also, making the colour of the textbox change to red, when a ship is severely damaged would be nice.

**Comment by silfumus** [comment]
add ?w=1 to the url to ignore whitespace changes.

**Comment by Zharay** [comment]
Got this in on my end, forgot to commit it. Thanks for pointing it out though.

**Pull request closed without merging**